examples/hitl/rearrange_v2/app_state_lobby.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Meta Platforms, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+from typing import Final, Optional
+
+from app_data import AppData
+from app_state_base import AppStateBase
+from app_states import create_app_state_rearrange
+
+from habitat_hitl.app_states.app_service import AppService
+
+START_SESSION_DELAY: Final[float] = 0.5
```

Comment:
Comment to explain the empirically chosen value? I see the comment below about disconnects, but still curious.